src/View/View.php

```diff
@@ -1259,7 +1259,7 @@ protected function _paths($plugin = null, $cached = true)
             $themePaths,
             $pluginPaths,
             $templatePaths,
-            [dirname(__DIR__) . DS . 'Template' . DS]
+            [CAKE . 'Template' . DS]
```

Comment:
Clearly we forgot to clean up DS here when it was moved to the app skeleton :blush: